10_Examples/will_onboarding1/include/will_onboarding1/Mather.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <chrono>
+#include <memory>
+#include <string>
+
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/int32.hpp"
+
+using namespace std::chrono_literals;
+using namespace std;
+using std::placeholders::_1;
+
```

Comment:
"using" is bad practice in a header file (some instances I may be guilty).

"using namespace" is REALLY bad practice.

Anything that includes this header (and any headers which include a header which include this) will get all of these propogated and basically removes namespacing all together, i.e. big trouble.